index.html

```diff
@@ -13597,8 +13597,9 @@ <h3>Presentational Roles Conflict Resolution</h3>
 		<ul>
 			<li>If an element is focusable, user agents MUST ignore the <code>presentation</code>/<code>none</code> role and expose the element with its implicit role, in order to ensure that the element is <a>operable</a>.</li>
 			<li>If a <a href="#mustContain">required owned element</a> has an explicit non-presentational role, user agents MUST ignore an inherited presentational role and expose the element with its explicit role. If the action of exposing the explicit role causes the accessibility tree to be malformed, the expected results are undefined.</li>
-			<li>If an element has global WAI-ARIA states or properties, user agents MUST ignore the <code>presentation</code> role and instead expose the element's implicit role. However, if an element has only non-global, role-specific WAI-ARIA states or properties, the element MUST NOT be exposed unless the presentational role is inherited and an explicit non-presentational role is applied.</li>
+			<li>If an element has <a>global</a> WAI-ARIA states or properties a, user agents MUST ignore the <code>presentation</code> role and instead expose the element's implicit role. However, if an element has only non-global, role-specific WAI-ARIA states or properties, the element MUST NOT be exposed unless the presentational role is inherited and an explicit non-presentational role is applied.</li>
 		</ul>
+		<p class="note">Some <a>global</a> WAI-ARIA states and properties are <a>prohibited</a> on some roles. These states and properties are still considered global states and properties for the purposes of Presentational Role Conflict resolution.</p>
```

Comment:
minor wording nit to not repeat "some" twice in the same sentence.  fully aware this is a personal opinion so please ignore if you would like.

```suggestion
		<p class="note">Some <a>global</a> WAI-ARIA states and properties are <a>prohibited</a> on certain roles. These states and properties are still considered global states and properties for the purposes of Presentational Role Conflict resolution.</p>
```